setup.py

```diff
@@ -10,6 +10,14 @@ def list_scripts():
     script_list = [f'{os.path.join(script_dir, f)}' for f in os.listdir(script_dir) if f.startswith('lstchain_')]
     return script_list
 
+entry_points = {}
+entry_points['console_scripts'] = [
+    'calc_camera_calibration = lstchain.tools.calc_camera_calibration:main',
+    'onsite_create_drs4_pedestal_file = scripts.onsite.onsite_create_drs4_pedestal_file:main',
+    'onsite_create_calibration_file = scripts.onsite.onsite_create_calibration_file:main',
+    'lstchain_data_create_pedestal_file = scripts.lstchain_data_create_pedestal_file:main'
+]
+
```

Comment:
`entry_points` are nice to transform functions into scripts but  I would suggest to not create an additional way of adding scripts (that requires adding each of them manually as entry point).
If you modify `list_scripts` this way, they will be automatically added:

```
def list_scripts():
    script_list = []
    script_dir = 'scripts'
    script_list.extend([f'{os.path.join(script_dir, f)}' for f in os.listdir(script_dir) if f.startswith('lstchain_')])
    script_dir = 'scripts/onsite'
    script_list.extend([f'{os.path.join(script_dir, f)}' for f in os.listdir(script_dir) if f.startswith('onsite')])
    return script_list
```

e.g. `lstchain_data_create_pedestal_file` is already a command line

Reply:
Hi @vuillaut,
the point is that I want to have the scripts as command line, so I don't what to see the .py at the end of the name (I think this is confusing for the user/shifter). Could we keep at least the onsite scripts as real command lines?

Reply:
Do you think users get this easily confused? 😄 
Either way, I don't mind, but I'd very much prefer to have a single implementation - that would require a bit of refactoring for the other scripts (that will need a main, and then be called as entry points) - do you think you can take care of it?
Also, I think you can automatize a bit to automatically add scripts following the schema `lstchain_***`and `onsite_***`

Reply:
Btw. The current scripts **are** command lines. The command name indeed ends with `.py`

Reply:
> Do you think users get this easily confused? 😄

yes, definitely ;-)

> Either way, I don't mind, but I'd very much prefer to have a single implementation - that would require a bit of refactoring for the other scripts (that will need a main, and then be called as entry points) - do you think you can take care of it?

Ok, I can, I think entry_points  is also the way ctapipe is doing, so let's be coherent with it

> Also, I think you can automatize a bit to automatically add scripts following the schema `lstchain_***`and `onsite_***`

Ok, I  try. 

I have some questions:
-> If we agree to put the config files in the "data" directory, should we change its name to "configurations" or similar?
-> My scrips are tools, I will changed their name with lstchain*, but I will keep them the tools directory, ok?
-> Should we approve this PR and then I do a new one with all these agreed changes ?
  (not to mix too many things)

Reply:
Ok let's go for entry points then 👍 

Yes sure keep the tools they are great! I actually think we should only make tools.
If you don't want to do that in this PR, please open issues or add actions in the board so we keep track of it.

Reply:
It depends on how long this PR takes to be approved ;-)
If it is fast I will do the changes in a new one, otherwise I will everything in this one.

Reply:
It's ok let's divide the work in smaller bits, then more people can contribute and review is easier :)

Reply:
Perfect, in this case, are there other changes to be done for this PR to be approved? (I will change the 100000 in the next PR, I think the problem is more complicated, in any case no sub-run will have more the 100000 events, so result is equivalent to None)